pandas/core/internals/api.py

```diff
@@ -0,0 +1,61 @@
+"""
```

Comment:
can you add a test for this model that asserts the exact name that we expose, similar to https://github.com/pandas-dev/pandas/blob/master/pandas/tests/api/test_api.py